sfm/message_consumer/management/commands/startconsumer.py

```diff
@@ -0,0 +1,19 @@
+import os
+from django.core.management.base import BaseCommand
+from message_consumer.sfm_ui_consumer import SfmUiConsumer
+from sfmutils.consumer import MqConfig, EXCHANGE
+
+HOST = "mq"
+QUEUE = "sfm_ui"
+ROUTING_KEYS = ["harvest.status.*", "harvest.status.*.*"]
+
+
+class Command(BaseCommand):
+    help = 'Starts the message consumer'
+
+    def handle(self, *args, **options):
+        username = os.environ['MQ_ENV_RABBITMQ_DEFAULT_USER']
```

Comment:
@justinlittman I thought this was the docker-ish way of passing in the environment values set in docker-compose?   Note that this is being done the same way in ```ui/rabbit.py```